calblueprint/chinese-newcomers
# conditionally render navigation stack

## Documentation
Conditionally render navigation stack based on user type (`admin` vs `normal_user`). This is done using an effect hook, where the `activeStack` is loaded based on 1) auth token exists — auth stack is loaded if not 2) active user type (i.e. admin vs normal user). 

## How to Review
- Login as an admin (i.e. email: admin@gmail.com, password: password) and verify that nav bar contains all four user flows.
- Login as a normal user (i.e. @celinechoiii ) and verify that nav bar contains only two flows (Feed and Signout).

## Related PRs

## Screenshots
![IMG_6748](https://user-images.githubusercontent.com/25868533/221445512-e634ca5d-3af3-4364-b3ed-46beb8699a25.PNG)
![IMG_6747](https://user-images.githubusercontent.com/25868533/221445513-8f5c4de0-d99b-4660-92ae-f1e4bacdb3e1.PNG)

CC: @surajr10 @themiachou8
